specification/sql/resource-manager/Microsoft.Sql/stable/2014-04-01/sql.core_legacy.json

```diff
@@ -108,6 +108,111 @@
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "name": "databaseName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the database for which setting the transparent data encryption applies."
+          },
+          {
+            "$ref": "#/parameters/TransparentDataEncryptionNameParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/TransparentDataEncryption"
+            },
+            "description": "The required parameters for creating or updating transparent data encryption."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/TransparentDataEncryption"
+            }
+          },
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/TransparentDataEncryption"
+            }
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "TransparentDataEncryption"
+        ],
+        "operationId": "TransparentDataEncryptions_Get",
+        "description": "Gets a database's transparent data encryption configuration.",
+        "x-ms-examples": {
+          "Get a database's transparent data encryption configuration": {
+            "$ref": "./examples/DatabaseTransparentDataEncryptionGet.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "name": "databaseName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the database for which the transparent data encryption applies."
+          },
+          {
+            "$ref": "#/parameters/TransparentDataEncryptionNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/TransparentDataEncryption"
+            }
+          }
+        }
+      }
+    },
     "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/transparentDataEncryption/{transparentDataEncryptionName}/operationResults": {
```

Comment:
not sure why put these API into sql.core.legacy.json. is there any other file something like transparentDataEncryption_legacy.json? and adding part is extract from where?

Reply:
The config for TransparentDataEncryption api was in the sql.core_legacy.json in the v4 tag, and in the v5 tag we were releasing a new version of this api so it's config was deleted from here. But since a few existing tests for this api are failing in .net sdk, we have decided to not release the new version. Hence adding this config back to revert to using an older and stable version of this api.